src/document-converter.ts

```diff
@@ -235,24 +235,22 @@ export class DocumentConverter {
 
     const outputProgram =
         recast.print(program, {quote: 'single', wrapColumn: 80, tabWidth: 2});
-    const deletedHtmlFileUrl =
-        ('./' + this.originalUrl) as ConvertedDocumentUrl;
-    return [
-      {
+
+    return {
+      originalUrl: this.originalUrl,
+      convertedUrl: this.convertedUrl,
+      convertedFilePath: this.originalUrl.replace('.html', '.js') as
```

Comment:
This looks potentially dangerous due to the unrestricted replace. Can you move to a central method to convert .html URLs to .js, and maybe only replace extensions?

Reply:
Good idea, I must have missed this when I was looking for other instances of fixHtmlExtension. Moving this into a helper method with the correct implementation.